openwisp_controller/config/migrations/0042_multiple_wireguard_tunnels.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 4.0.5 on 2022-06-24 06:31
+
+from django.db import migrations
+
+from ..migrations import get_swapped_model
+
+
+def get_wireguard_and_vxlan_wireguard_templates(apps):
+    template_model = get_swapped_model(apps, 'config', 'Template')
+    return template_model.objects.filter(type='vpn', vpn__backend__contains='Wireguard')
+
+
+def allow_multiple_wireguard_tunneling(apps, schema_editor):
+    templates = get_wireguard_and_vxlan_wireguard_templates(apps).iterator()
+    for template in templates:
+        config = template.config
+        interfaces = config['interfaces']
+        vpn_id = template.vpn.pk.hex
+        changed = False
+        for interface in interfaces:
+            if interface['private_key'] not in [
```

Comment:
hey, this risks to fail if `private_key` is not defined (eg, if it's another interface)